WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -1,389 +1,57 @@
-        mApplicationLifecycleMonitor = new ApplicationLifecycleMonitor();
-        ProcessLifecycleOwner.get().getLifecycle().addObserver(this);
-
-        // Make the UploadStarter observe the app process so it can auto-start uploads
-        mUploadStarter.activateAutoUploading((ProcessLifecycleOwner) ProcessLifecycleOwner.get());
-
-        initAnalytics(SystemClock.elapsedRealtime() - startDate);
-
-        createNotificationChannelsOnSdk26();
-
-        // Allows vector drawable from resources (in selectors for instance) on Android < 21 (can cause issues
-        // with memory usage and the use of Configuration). More information: http://bit.ly/2H1KTQo
-        // Note: if removed, this will cause crashes on Android < 21
-        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
-        AppThemeUtils.Companion.setAppTheme(this);
-
-        // verify media is sanitized
-        sanitizeMediaUploadStateForSite();
-
-        // remove expired lists
-        mDispatcher.dispatch(ListActionBuilder.newRemoveExpiredListsAction(new RemoveExpiredListsPayload()));
-
-        // setup the Credentials Client so we can clean it up on wpcom logout
-        mCredentialsClient = new GoogleApiClient.Builder(this)
-                .addConnectionCallbacks(new GoogleApiClient.ConnectionCallbacks() {
-                    @Override
-                    public void onConnected(@Nullable Bundle bundle) {
-                    }
-
-                    @Override
-                    public void onConnectionSuspended(int i) {
-                    }
-                })
-                .addApi(Auth.CREDENTIALS_API)
-                .build();
-        mCredentialsClient.connect();
-
-        initWorkManager();
-
-        // Enqueue our periodic upload work request. The UploadWorkRequest will be called even if the app is closed.
-        // It will upload local draft or published posts with local changes to the server.
-        UploadWorkerKt.enqueuePeriodicUploadWorkRequestForAllSites();
-
-        mSystemNotificationsTracker.checkSystemNotificationsState();
-        ImageEditorInitializer.Companion.init(
-                mImageManager,
-                mImageEditorTracker,
-                mImageEditorFileUtils,
-                mAppScope
-        );
-
-        initEmojiCompat();
-        mStoryNotificationTrackerProvider = new StoryNotificationTrackerProvider();
-        mStoryMediaSaveUploadBridge.init(this);
-        ProcessLifecycleOwner.get().getLifecycle().addObserver(mStoryMediaSaveUploadBridge);
-
-        mExPlat.forceRefresh();
-
-        mDebugCookieManager.sync();
-    }
-
-    protected void initWorkManager() {
-        androidx.work.Configuration config =
-                (new androidx.work.Configuration.Builder()).setWorkerFactory(mWordPressWorkerFactory).build();
-        WorkManager.initialize(this, config);
+        return AppInitializer.Companion.getBitmapCache();
```

Comment:
Good catch! 👍🏻
I'm removing `@JvmStatic` annotation from functions instead of removing `Companion`.  Because we won't need `@JvmStatic` when we convert `WordPress` to Kotlin.